oca_port/migrate_addon.py

```diff
@@ -6,8 +6,8 @@
 import tempfile
 import urllib.parse
 
-from . import misc
-from .misc import bcolors as bc
+from .utils import misc, git as g
```

Comment:
IMO `git` is already short enough ;)

Reply:
then, why not `github as gh`? :stuck_out_tongue:

Reply:
It was to not overlapping by mistake the third party package `git` (GitPython) and the `utils.git` one of this package.